modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -127,6 +132,106 @@ def tx_kill(self, **kwargs):
         res = self.__parse_tx_list(output)
         return res if res else output
 
+    def validate_indexes(self, check_assert: bool = None):
+        """
+        Validate indexes.
+        If check is true, will be return
+        """
+        data = self.__run("--cache validate_indexes")
+
+        if check_assert is not None:
+            assert (('no issues found.' in data) == check_assert), data
+
+        return data
```

Comment:
any reason to have this return?

Reply:
I think the control_utility should return a result

Reply:
fixed